docs/getting_started/install-on-mac-os-and-windows.md

```diff
@@ -0,0 +1,98 @@
+# Installation on macOS or Windows
+
+This page explains how to install [[= product_name =]] on macOS or Windows.
+
+!!! caution
+
+    This procedure is **for development purposes only**.
+    Installing [[= product_name =]] for production purposes is supported only on Linux.
+
+    If you want to use [[= product_name =]] in the production environment, see [Installing [[= product_name =]]](../getting_started/install_ez_platform.md).  
+
+### Prepare work environment
+
+To install [[= product_name =]], you need a stack with MySQL and PHP.
+Additionally, you need [Node.js](https://nodejs.org/en/) and [Yarn](https://yarnpkg.com/lang/en/docs/install/) for asset management.
+If you want to use a web server, you need to install it as well:
+
+- For Windows: Apache
+- For macOS: Apache/nginx
+
+The instructions below assume you are using Apache.
+
+??? "Windows"
+
+    Locate the `php.ini` file and open it in a text editor.
```

Comment:
We sould put the steps in a numbered list (with substeps) for clarity. We can discuss it later